Question protocol with supporting components and config changes (#20)
DFE-Digital/bat-design-history: app/posts/apply-for-teacher-training/2020-01-16-question-protocol-for-pilot.md

    }],
    what: ["Answer not submitted as part of application"]
  }
}) }}

{{ appProtocol({
  title: "Training provider",
  items: {
    question: "Which training provider are you applying to?",
    required: "Yes",
    api: {
      param: "course.provider_code",
      href: "https://www.apply-for-teacher-training.education.gov.uk/api-docs/reference/#course-object"
    },
    why: [{
      who: ["service"],
      what: "uses this to know which provider to send the completed application"
    }, {
      who: ["hesa"],
      what: "requires this to identify source of student record"
    }, {
      who: ["dfe"],
      what: "uses this for reporting"
    }],
    what: ["Retrievable via API"],
    accuracy: ["Candidates can only choose from a pre-populated list of providers which is driven by data from the Find service"]
  }
}) }}

{{ appProtocol({
  title: "Course",
  items: {
    question: "Which course are you applying to?",
    required: "Yes",
    api: {
      param: "course.course_code",
      href: "https://www.apply-for-teacher-training.education.gov.uk/api-docs/reference/#course-object"
    },
    why: [{
      who: ["provider"],

>       what: "uses this to determine funding"

Reviewer: ```suggestion
      what: "uses this to know which course is being applied to, and to determine funding"
```

    }],
    what: ["Appears in provider-facing interface and retrievable via API"],
    accuracy: ["Providers ask for proof of identification"],
    notes: [
      "Ideally this field would be combined with given name but recorded separately to support legacy student record systems.",
      "HESA also ask for ‘Immediately prior surname’ to “to allow matching of records where a student has changed surname” but we don’t capture this information.",
      "Should we allow changes post-submission?"
    ]
  }
}) }}

{{ appProtocol({
  title: "Date of birth",
  items: {
    question: "Date of birth",
    required: "Yes",
    api: {
      param: "candidate.date_of_birth",
      href: "https://www.apply-for-teacher-training.education.gov.uk/api-docs/reference/#candidate-object"
    },
    why: [{
      who: ["provider"],
      what: "use this when performing a DBS check"
    }, {
      who: ["hesa"],
      what: "needs this to be in the format YYYY-MM-DD in order to link to other records (e.g. higher education, wider than just ITT)"
    }, {
      who: ["dfe"],

>       what: "uses this for record matching and statistical analysis, e.g. looking at application rates of over 35 year olds (something the minister has historically been interested in)"

Reviewer: GOV.UK style drops the dots from etc, eg, ie – but also recommends against them:
https://www.gov.uk/guidance/style-guide/a-to-z-of-gov-uk-style#eg-etc-and-ie

Author: Updated and replaced all occurrences of e.g. and i.e.

    }],
    what: ["Appears in provider-facing interface and retrievable via API"],
    accuracy: ["Providers ask for proof of identification"],

>     notes: ["We need to collect year, but do we need to show it to providers?"]

Reviewer: ```suggestion
    notes: ["We need to collect year, but do we need to show it to providers? And if so, at what point?"]
```

  }
}) }}

{{ appProtocol({
  title: "Nationality",
  items: {
    question: "Nationality",
    required: "Yes",
    api: {
      param: "candidate.nationality[]",
      href: "https://www.apply-for-teacher-training.education.gov.uk/api-docs/reference/#candidate-object"
    },
    why: [{
      who: ["provider"],
      what: "may use this to assess if a candidate is [eligibile for student finance](https://www.gov.uk/student-finance/who-qualifies)"
    }, {
      who: ["hesa"],
      what: "requires this “to respond to Parliamentary Questions, Written Questions, FOI requests on complex citizenship/nationality issues. To be able to perform analysis on trainee demographics at the start of an academic year.”"
    }],
    what: ["Appears in provider-facing interface and retrievable via API"],
    accuracy: ["Providers ask for proof of identification"],

>     notes: ["We need to collect year, but do we need to show it to providers?"]

Reviewer: Copy paste error.

Probably needs notes about:
- multi nationality (2 or more)
- no nationality (stateless, refugee)
- unknown nationality

  }
}) }}

{{ appProtocol({
  title: "Second nationality",
  items: {

>     question: "Add another nationality → Second nationality",

Reviewer: Nice.

    required: "No",
    api: {
      param: "candidate.nationality[]",
      href: "https://www.apply-for-teacher-training.education.gov.uk/api-docs/reference/#candidate-object"
    },
    what: ["Appears in provider-facing interface and retrievable via API"],

>     notes: ["We needn’t ask for additional nationalities; dual nationals can enter their British or EEA-derived nationality"]

Reviewer: There's also something here about multi nationals having to make a choice about which other nationality/nationalities they show. We don't help them with this. Adding all might simplify.

Author: Updated note.

  }
}) }}

{{ appProtocol({
  title: "✘ Residency status",
  items: {
    question: "What is your residence status?",
    required: "Yes (if country of nationality is not within the UK or EEA)",
    api: {
      param: "candidate.uk_residency_status",
      href: "https://www.apply-for-teacher-training.education.gov.uk/api-docs/reference/#candidate-object"
    },
    why: [{
      who: ["provider"],
      what: "may use this field to assess if a candidate is [eligibile for student finance](https://www.gov.uk/student-finance/who-qualifies)"
    }, {
      who: ["hesa"],
      what: "requires domicile to “distinguish between home and overseas students and to disaggregate students by origin. To obtain the numbers of students domiciled in particular EC countries and elsewhere abroad, and hence calculate participation rates. To forecast the demand for student allowances and grants. To record cross-border flows.”"
    }],
    what: ["Appears in provider-facing interface and retrievable via API"],
    accuracy: ["Providers ask for documentation"],

>     notes: ["In addition to status, do we also need to ask for the date they entered the country? This might be relevant if an applicant has a Tier 5 Visa (from Australia or New Zealand, where people can work for up to 2 years in the UK). This may also impact funding decisions"]

Reviewer: ```suggestion
    notes: ["In addition to status, do we also need to ask for the date they entered the country? This might be relevant if an applicant has a Tier 5 Visa (from Australia or New Zealand, where people can work for up to 2 years in the UK). Date they entered the country can also impact funding."]
```

    api: {
      param: "candidate.english_language_qualifications",
      href: "https://www.apply-for-teacher-training.education.gov.uk/api-docs/reference/#candidate-object"
    },
    why: [{
      who: ["unknown"],
      what: "Further research required"
    }],
    what: ["Appears in provider-facing interface and retrievable via API"],
    notes: ["Does having an English language GCSE (or equivalent) mean we need to ask this question? Does an English GCSE indicate fluency?"]
  }
}) }}

{{ appProtocol({
  title: "Other languages spoken",
  items: {
    question: "If you are bilingual or very familiar with languages other than English, you can tell us about them here",
    required: "No",
    api: {
      param: "candidate.other_languages",
      href: "https://www.apply-for-teacher-training.education.gov.uk/api-docs/reference/#candidate-object"
    },
    why: [{
      who: ["candidate"],
      what: "may want to demonstrate language skills, especially if looking to teach a language subject or teach in school with children whose main language isn’t English"
    }],
    what: ["Appears in provider-facing interface and retrievable via API"],

>     accuracy: ["Assesed at interview"],

Reviewer: ```suggestion
    accuracy: ["Assessed at interview"],
```

>     notes: ["SCITTs/SDs we've recently spoken to haven’t expressed a need for this, but it’s something candidates sometimes like to mention"]

Reviewer: Something about being pertinent to modern languages courses?

Author: That’s included in the who/what part of the protocol. Have added ‘modern’.

    required: "Yes",
    api: {
      param: "contact_details.email",
      href: "https://www.apply-for-teacher-training.education.gov.uk/api-docs/reference/#contactdetails-object"
    },
    why: [{
      who: ["service"],
      what: "requires this to send candidates a ‘magic link’ to sign back in, and to send updates about their application"
    }, {
      who: ["provider"],
      what: "requires this to contact candidates to arrange interviews and send communications relevant to their application"
    }, {
      who: ["hesa"],
      what: "requires this as the “DfE requires this to communicate with newly qualified teachers”"
    }],
    what: ["Appears in provider-facing interface and retrievable via API"],
    accuracy: [
      "Address format is validated",
      "Service requires a verified email address to sign in"
    ]
  }
}) }}

{{ appProtocol({
  title: "Phone number",
  items: {
    question: "Phone number",
    required: "Yes",
    api: {
      param: "contact_details.phone_number",
      href: "https://www.apply-for-teacher-training.education.gov.uk/api-docs/reference/#contactdetails-object"
    },
    why: [{
      who: ["provider"],

>       what: "may use this to contact a candidate"

Reviewer: We also used it for research I think.

Author: Updated.

    }],
    what: ["Appears in provider-facing interface and retrievable via API"],

>     accuracy: ["❓"],

Reviewer: We validate format.

But we also allow international numbers without their country code (this has happened)

Author: Updated note.

    notes: ["Do providers actually need a phone number? Some candidates might not want to be contacted this way."]
  }
}) }}

{{ appProtocol({
  title: "Address",
  items: {
    question: "What is your address?",
    required: "Yes",
    api: {
      param: "contact_details.address_line1
        contact_details.address_line2
        contact_details.address_line3
        contact_details.address_line4
        contact_details.postcode
        contact_details.country",
      href: "https://www.apply-for-teacher-training.education.gov.uk/api-docs/reference/#contactdetails-object"
    },
    why: [{
      who: ["provider"],
      what: "may use this to write to candidates"
    }, {
      who: ["provider"],
      what: "may use this to advise candidates on travel and suitability of different campuses"
    }, {
      who: ["provider"],

>       what: "may use this to determine the seriousness of an application (e.g. proximity to their institution) and to advise candidates on travel and suitability of different campuses"

Reviewer: Seems like something that _could_ be removed, worth investigating.

    }, {
      who: ["dfe"],
      what: "uses this for demographic reporting"
    }],
    what: ["Appears in provider-facing interface and retrievable via API"],

>     accuracy: ["In the future we can use a postcode look-up database to fill in UK addresses"],

Reviewer: Also postcode validation

Author: Updated notes.

      href: "https://www.apply-for-teacher-training.education.gov.uk/api-docs/reference/#workexperiences-object"
    },
    why: [{
      who: ["service"],
      what: "uses this label to distinguish between different jobs when reviewing answers"
    }, {
      who: ["provider"],
      what: "uses this to build a picture of a candidate’s background"
    }],
    what: ["Appears in provider-facing interface and retrievable via API"],
    accuracy: ["We don’t directly, but a reference from an employer may collaborate some of this information"]
  }
}) }}

{{ appProtocol({
  title: "Job - Employer",
  items: {
    question: "Name of employer",
    required: "Yes",
    api: {
      param: "work_experience.jobs[].organisation_name",
      href: "https://www.apply-for-teacher-training.education.gov.uk/api-docs/reference/#workexperiences-object"
    },
    why: [{
      who: ["service"],
      what: "uses this label to distinguish between different jobs when reviewing answers"
    }, {
      who: ["provider"],
      what: "uses this to build a picture of a candidate’s background"
    }],
    what: ["Appears in provider-facing interface and retrievable via API"],

>     accuracy: ["We don’t directly, but a reference from an employer may collaborate some of this information"]

Reviewer: Notes about being self employed?

Author: Updated note.

  }
}) }}

{{ appProtocol({
  title: "Job - Time commitment",
  items: {

>     question: "Was this job full-time or part-time?",

Reviewer: ```suggestion
    question: "Was this job full time or part time?",
```

    required: "Yes",
    api: {
      param: "work_experience.jobs[].commitment",
      href: "https://www.apply-for-teacher-training.education.gov.uk/api-docs/reference/#workexperiences-object"
    },
    why: [{
      who: ["provider"],
      what: "uses this to build a picture of a candidate’s background"
    }],
    what: ["Appears in provider-facing interface and retrievable via API"],
    accuracy: ["We don’t directly, but a reference from an employer may collaborate some of this information"],

>     notes: ["Feedback from HEIs has suggested hours per week is more useful (this is what UCAS asks)"]

Reviewer: ```suggestion
    notes: ["Feedback from HEIs has suggested hours per week is more useful (this is what UCAS asks). We might extend to include details of working pattern for part time jobs."]
```

  }
}) }}

{{ appProtocol({
  title: "Job - Period of employment",
  items: {
    question: "Start date
      End date",
    required: "Yes",
    api: {
      param: "work_experience.jobs[].start_date
      work_experience.jobs[].end_date",
      href: "https://www.apply-for-teacher-training.education.gov.uk/api-docs/reference/#workexperiences-object"
    },
    why: [{
      who: ["service"],
      what: "uses this to calculate any gaps in work history"
    }, {
      who: ["provider"],
      what: "uses this to build a picture of a candidate’s background"
    }],
    what: ["Appears in provider-facing interface and retrievable via API"],

>     accuracy: ["We don’t directly, but a reference from an employer may collaborate some of this information"]

Reviewer: We check range is valid

Missing notes about only asking for months and not specific days?

Author: Updated.

  }
}) }}

{{ appProtocol({
  title: "Job - Skills and experience",
  items: {
    question: "Skills and experience relevant to teaching you gained in this role",
    required: "Yes",
    api: {
      param: "work_experience.jobs[].description",
      href: "https://www.apply-for-teacher-training.education.gov.uk/api-docs/reference/#workexperiences-object"
    },
    why: [{
      who: ["candidate"],
      what: "may use this to demonstrate subject knowledge gained in the workplace"
    }, {
      who: ["provider"],
      what: "uses this to build a picture of a candidate’s background"
    }],
    what: ["Appears in provider-facing interface and retrievable via API"],

>     accuracy: ["We don’t directly, but a reference from an employer may collaborate some of this information"]

Reviewer: Notes – should we indicate why this field is different to a more standard text field for "job description"?

      who: ["provider"],
      what: "uses this to build a picture of a candidate’s background, and answer questions about any gaps in work history"
    }],
    what: ["Appears in provider-facing interface and retrievable via API"],
    accuracy: ["We don’t directly, but a reference from an employer may collaborate some of this information"],
    notes: ["For v1 of the API, any information about breaks will be appended to the above named parameter. When we start collecting date information around breaks, we'll include those in the text field too."]
  }
}) }}

{{ appProtocol({
  title: "Break - Reason for work break",
  items: {
    question: "Enter reasons for break in work history",
    required: "Yes (if any gaps appear in work history)",
    api: {
      param: "work_experience.work_history_break_explanation",
      href: "https://www.apply-for-teacher-training.education.gov.uk/api-docs/reference/#workexperiences-object"
    },
    why: [{
      who: ["provider"],
      what: "uses this to build a picture of a candidate’s background, and answer questions about gaps in employment history"
    }],
    what: ["Appears in provider-facing interface and retrievable via API"],

>     accuracy: ["❓"],

Reviewer: Assessed at interview?

    required: "No",
    api: {
      param: "work_experience.volunteering[].organisation_name",
      href: "https://www.apply-for-teacher-training.education.gov.uk/api-docs/reference/#workexperiences-object"
    },
    why: [{
      who: ["service"],
      what: "uses this label to distinguish between different roles"
    }, {
      who: ["provider"],
      what: "uses this to build a picture of a candidate’s background"
    }],
    what: ["Appears in provider-facing interface and retrievable via API"],
    accuracy: ["Reference may collaborate some of this information"]
  }
}) }}

{{ appProtocol({
  title: "Role - Dates of volunteering",
  items: {
    question: "Start date
      End date",
    required: "No",
    api: {
      param: "work_experience.volunteering[].start_date
      work_experience.volunteering[].end_date",
      href: "https://www.apply-for-teacher-training.education.gov.uk/api-docs/reference/#workexperiences-object"
    },
    why: [{
      who: ["provider"],
      what: "uses this to build a picture of a candidate’s background"
    }],
    what: ["Appears in provider-facing interface and retrievable via API"],

>     accuracy: ["Reference may collaborate some of this information"]

Reviewer: Again, we check the range is a valid one, and we don't care about day of the month.

  }
}) }}

{{ appProtocol({
  title: "Role - Time commitment and responsibilities",
  items: {
    question: "Enter details of your time commitment and responsibilities",
    required: "No",
    api: {
      param: "work_experience.volunteering[].description",
      href: "https://www.apply-for-teacher-training.education.gov.uk/api-docs/reference/#workexperiences-object"
    },
    why: [{
      who: ["candidate"],

>       what: "can use this to demonstrate what they learnt during school experience"

Reviewer: ```suggestion
      what: "can use this to demonstrate what they learnt during their unpaid experience"
```

    }, {
      who: ["provider"],
      what: "uses this to build a picture of a candidate’s background"
    }],
    what: ["Appears in provider-facing interface and retrievable via API"],
    accuracy: ["Reference may collaborate some of this information"]
  }
}) }}

{{ appProtocol({
  title: "Role - Role involved working with children",
  items: {

>     question: "Did this job involve working in a school or with children?",

Reviewer: Eek, should this say "role" (it's like this on live too)?

Author: Made a note to make a card!

    required: "No",
    api: {
      param: "work_experience.volunteering[].working_with_children",
      href: "https://www.apply-for-teacher-training.education.gov.uk/api-docs/reference/#workexperiences-object"
    },
    why: [{
      who: ["candidate"],
      what: "may use this to indicate experience working with children"
    }, {
      who: ["provider"],
      what: "uses this to build a picture of a candidate’s suitability"
    }],
    what: ["Appears in provider-facing interface and retrievable via API"],
    accuracy: ["Reference may collaborate some of this information"],

>     notes: ["We’ve allowed people to include volunteering roles that don't involve kids – this could be clearer though."]

Reviewer: ```suggestion
    notes: ["We’ve allowed people to include volunteering roles that don't involve kids – this needs to be clearer."]
```

  }
}) }}

## Training with a disability

This question in the form, as described below, was removed from the initial pilot for launch. It will return once additional research in this area has been completed.

Candidates can update their answers before (and 5 working days after) submitting their application.

{{ appProtocol({
  title: "✘ Candidate wants to disclose a disability",
  items: {
    question: "Do you want to disclose a disability?",
    required: "Yes",
    why: [{
      who: ["service"],
      what: "uses the answer to show/hide textarea"
    }],

>     what: ["Answer not submitted as part of application"]

Reviewer: Isn't the yes/no used by providers?

Author: Updated.

  }
}) }}

{{ appProtocol({
  title: "✘ Disability disclosure",
  items: {
    question: "Tell us about your disability",
    required: "No",
    api: {
      param: "candidate.disability_disclosure",
      href: "https://www.apply-for-teacher-training.education.gov.uk/api-docs/reference/#candidate-object"
    },
    why: [{
      who: ["provider"],
      what: "may use this to support the candidate at interview (and training if accepted onto course)"
    }],
    what: ["Appears in provider-facing interface and retrievable via API"],

>     accuracy: ["Assessed at or before interview"],

Reviewer: Not sure assessed is the right word here.

Author: Reworded.

    notes: [
      "Does a provider or someone else look at this and make any reasonable adjustments? Is there a process to work out what adjustments can be made or not? Research currently being undertaken."
    ],

>     history: ["/apply-for-teacher-training/training-with-a-disability"]

Reviewer: 💘

  }
}) }}

## Degrees

Candidates can update their answers before (and 5 working days after) submitting their application.

{{ appProtocol({
  title: "Qualification - type",
  items: {
    question: "Type of degree",
    required: "First degree: Yes
      Additional degree(s): No",
    api: {
      param: "qualifications.degrees[].qualification_type",
      href: "https://www.apply-for-teacher-training.education.gov.uk/api-docs/reference/#qualification-object"
    },
    why: [{
      who: ["provider"],

>       what: "needs to ensure candidate holds a first degree of a UK higher education institution or equivalent qualification"

Reviewer: Suggest linking to:
https://www.gov.uk/government/publications/initial-teacher-training-criteria/initial-teacher-training-itt-criteria-and-supporting-advice#c12-degree-criteria

Author: Done.

      href: "https://www.apply-for-teacher-training.education.gov.uk/api-docs/reference/#qualification-object"
    },
    why: [{
      who: ["provider"],
      what: "wants to see that a candidate can demonstrate subject and curriculum knowledge"
    }, {
      who: ["hesa"],
      what: "requires this to “provide data on the qualifications of teachers.”"
    }],
    what: ["Appears in provider-facing interface and retrievable via API"],
    accuracy: ["Providers will ask to see hard copies for any qualifications awarded"]
  }
}) }}

{{ appProtocol({
  title: "Qualification - grade",
  items: {
    question: "Enter your degree grade",
    required: "First degree: Yes
      Additional degree(s): No",
    api: {
      param: "qualifications.degrees[].grade",
      href: "https://www.apply-for-teacher-training.education.gov.uk/api-docs/reference/#qualification-object"
    },
    why: [{
      who: ["hesa"],
      what: "requires this to “provide data on the qualifications of teachers.”"
    }],
    what: ["Appears in provider-facing interface and retrievable via API"],
    accuracy: ["Providers will ask to see hard copies for any qualifications awarded"],

>     notes: ["Not all postgraduate degrees have grades"]

Reviewer: ```suggestion
    notes: ["Not all postgraduate degrees have grades, for example a PhD"]
```

  }
}) }}

{{ appProtocol({
  title: "Qualification - year",
  items: {
    question: "What year did you graduate?",
    required: "First degree: Yes
      Additional degree(s): No",
    api: {
      param: "qualifications.degrees[].award_year",
      href: "https://www.apply-for-teacher-training.education.gov.uk/api-docs/reference/#qualification-object"
    },
    why: [{
      who: ["hesa"],
      what: "requires this to “create the student's DfE record.”"
    }],
    what: ["Appears in provider-facing interface and retrievable via API"],
    accuracy: ["Providers will ask to see hard copies for any qualifications awarded"]
  }
}) }}

{{ appProtocol({
  title: "Qualification - institution",
  items: {
    question: "Institution where you studied",
    required: "First degree: Yes
      Additional degree(s): No",
    api: {
      param: "qualifications.degrees[].institution_details",
      href: "https://www.apply-for-teacher-training.education.gov.uk/api-docs/reference/#qualification-object"
    },
    why: [{
      who: ["provider"],

>       what: "may be interested in calibre of the institution studied at"

Reviewer: A dance provider mentioned they'd use this information to make sure they didn't send a trainee teacher to one of the schools they had previously trained at.

Author: Added this note.

    }, {
      who: ["hesa"],
      what: "requires this to “create the student's DfE record.”"
    }],
    what: ["Retrievable via API"],
    accuracy: ["Providers will ask to see hard copies for any qualifications awarded"],
    notes: [
      "This may be a want, more than a need.",
      "Not currently shown to providers"
    ]
  }
}) }}

{{ appProtocol({
  title: "✘ Qualification - (non-UK) country",
  items: {
    question: "Country (if outside UK)",
    required: "Yes (if degree not awarded in UK)",
    api: {
      param: "qualifications.degrees[].awarding_body",
      href: "https://www.apply-for-teacher-training.education.gov.uk/api-docs/reference/#qualification-object"
    },
    why: [{
      who: ["provider"],

>       what: "uses this to perform a NARIC check (they need confirmation from a the institution that they’ve actually obtained the degree they say they have)"

Reviewer: Link to NARIC?

Author: Done!

      what: "needs to look for evidence that a qualification is of a standard equivalent to a UK degree"
    }, {
      who: ["candidate"],
      what: "needs to provide evidence that their international qualification demonstrates a similar level and breadth as the equivalent UK qualification"
    }, {
      who: ["candidate"],
      what: "may already have a statement of comparability from NARIC"
    }],
    what: ["Appears in provider-facing interface and retrievable via API"],
    accuracy: ["Providers will ask to see hard copies for any qualifications awarded"],
    notes: ["Candidates can request a statement of comparability from NARIC, or speak to a TTA who can certify equivalence"]
  }
}) }}

## Maths GCSE or equivalent

Candidates can update their answers before (and 5 working days after) submitting their application.

{{ appProtocol({
  title: "Qualification - type",
  items: {
    question: "Type of qualification",
    required: "Yes",
    api: {
      param: "qualifications.gcses[].qualification_type",
      href: "https://www.apply-for-teacher-training.education.gov.uk/api-docs/reference/#qualification-object"
    },
    why: [{
      who: ["provider"],

>       what: "needs to ensure candidates have achieved a standard equivalent to a grade 4 in GCSE Maths"

Reviewer: Suggest linking to https://www.gov.uk/government/publications/initial-teacher-training-criteria/initial-teacher-training-itt-criteria-and-supporting-advice#c11-gcse-standard-equivalent

>       what: "needs to ensure candidates have achieved a standard equivalent to a grade 4 in GCSE Maths"

Reviewer: ```suggestion
      what: "needs to ensure candidates for primary and secondary courses have achieved a standard equivalent to a grade 4 in GCSE Maths"
```

    api: {
      param: "qualifications.gcses[].equivalency_details",
      href: "https://www.apply-for-teacher-training.education.gov.uk/api-docs/reference/#qualification-object"
    },
    why: [{
      who: ["candidate"],
      what: "needs to provide evidence that their international qualification demonstrates a similar level and breadth as the equivalent UK qualification"
    }, {
      who: ["candidate"],
      what: "may already have a statement of comparability from NARIC"
    }, {
      who: ["provider"],
      what: "needs to look for evidence that a qualification is of a standard equivalent to GCSE grade 4, not only in terms of its level, but also in terms of its breadth. Details of qualification can help determine if equivalent to GCSE"
    }],
    what: ["Appears in provider-facing interface and retrievable via API"],
    accuracy: ["Providers will ask to see hard copies for any qualifications awarded"],
    notes: ["Candidates can request a statement of comparability from NARIC, or speak to a TTA who can certify equivalence"]
  }
}) }}

## English GCSE or equivalent

Candidates can update their answers before (and 5 working days after) submitting their application.

> Providers also need further evidence of a breadth of achievement in English where applicants have achieved a GCSE grade 4 or above *in English literature only*. We don’t currently make this distinction in the service.

Reviewer: I thought it was English language? Interesting that the ITT criteria don't directly reference this.

Author: Yeah, guidance could be clearer and explicitly state English language. It does add further detail in the guidance:

> Providers should look for further evidence of a breadth of achievement in English where applicants have achieved a GCSE grade 4 or above in English literature only.

    }],
    what: ["Appears in provider-facing interface and retrievable via API"],
    accuracy: [
      "Providers will ask to see hard copies for any qualifications awarded",
      "In the future we can check any post-14 qualifications (awarded after 2006 in England, Wales and Northern Ireland) against the Learning Records Service"]
  }
}) }}

{{ appProtocol({
  title: "Qualification - grade",
  items: {
    question: "What grade is your English qualification?",
    required: "Yes (if applying for a Primary course)",
    api: {
      param: "qualifications.gcses[].grade",
      href: "https://www.apply-for-teacher-training.education.gov.uk/api-docs/reference/#qualification-object"
    },
    why: [{
      who: ["provider"],
      what: "needs to ensure candidates have achieved a standard equivalent to a grade 4 in a GCSE science subject"
    }],
    what: ["Appears in provider-facing interface and retrievable via API"],
    accuracy: [
      "Providers will ask to see hard copies for any qualifications awarded",
      "In the future we can check any post-14 qualifications (awarded after 2006 in England, Wales and Northern Ireland) against the Learning Records Service"],

>     notes: ["Do candidates need to specify which subjects they obtained each grade in? (e.g. double or triple award science)?"]

Reviewer: ITT criteria says:
> a standard equivalent to a grade 4 in the GCSE examination _in a science subject_

Which I read as "any".

But, HEI providers are asking to know which subjects.

Author: Updated note.

  }
}) }}

{{ appProtocol({
  title: "Referee - email address",
  items: {
    question: "Email address",
    required: "Yes",
    api: {
      param: "referees[].email",
      href: "https://www.apply-for-teacher-training.education.gov.uk/api-docs/reference/#reference-object"
    },
    why: [{
      who: ["service"],
      what: "uses this to contact a referee and request a reference"
    }, {
      who: ["provider"],
      what: "use this to validate that a reference is from an official work source (with the domain name used as a proxy)"
    }],
    what: ["Appears in support interface. If the referee provides a reference, appears in the provider-facing interface and retrievable via API"],
    accuracy: [
      "Address format is validated",
      "If email to the nominated referee bounces, we can request correct details from the candidate"
    ],
    notes: ["Should providers be assessing based on this information? We don’t surface to candidates that they need to submit professional email addresses. How do we accommodate candidates (career changers, people returning to the workplace, etc.) who may struggle to provide such information?"]
  }
}) }}

{{ appProtocol({

>   title: "Referee - relation

Reviewer: Referee might corroborate (thought we don't currently show to them)

Author: Updated.

    }],
    what: ["Appears in provider-facing interface and retrievable via API"],
    accuracy: ["Providers ask for proof of identification"],
    notes: [

>       "Candidate’s may have multiple nationalities (2 or more). We currently account for this in a subsequent question",

Reviewer: Grocers apostrophe.

Author: Fixed on master.
